### ISSUE TITLE
Help: How to reduce Disk size to erase and reinstall Mac

### ISSUE THREAD

**Issue by manjotsidhu** [sickcodes/Docker-OSX#243]
Following the Post-boot instructions: https://github.com/sickcodes/Docker-OSX#additional-boot-instructions-for-when-you-are-creating-your-container. Erase button is disabled for me, and I don't know how to reduce the disk size. I tried Partitioning/Adding Volume but both failed showing me error message.

**Comment by sickcodes** [comment]
We are using qemu images so the disk dynamically changes size. You cannot change the size of the booted partition.

Boot into the Recovery mode.

Can you show screenshot of which stage you are at?

**Comment by manjotsidhu** [comment]
> We are using qemu images so the disk dynamically changes size. You cannot change the size of the booted partition.
> 
> Boot into the Recovery mode.
> 
> Can you show screenshot of which stage you are at?

I have downloaded the prebuilt container and using the [Container Creation Examples](https://github.com/sickcodes/Docker-OSX#container-creation-examples). I have successfully booted that container but since README also points to post-boot instructions. I am currently stuck there. I can see 3 partitions in my Disk Utility, 2 are 214GB in which I am not getting uninstall option.
